sites/docs/src/content/docs/nf-core-tools/modules/lint.md

```diff
@@ -7,11 +7,16 @@ weight: 80
 
 Run the `nf-core modules lint` command to check modules in the current working directory (pipeline or nf-core/modules clone) against nf-core guidelines.
 
-Use the `--all` flag to run linting on all modules found. Use `--dir <pipeline_dir>` to specify another directory than the current working directory.
+- Use the `--all` flag to run linting on all modules found.
+- Use `--dir <pipeline_dir>` to specify another directory than the current working directory.
+- Use the `--fix` flag to fix all possible linting errors:
+  - Write the correct input and output channels and channel structure. This will try to use the information existing in your meta.yml, but correct the structure of the `meta.yml` file if needed. Make sure to check that the added changes are correct and update `description`, `pattern` and `type` accordingly.
```

Comment:
do you mean main.nf instead of meta.yml in the second sentence here?

Reply:
No, meta.yml is correct. I want to say that if you have existing information there, it will reuse it, instead of deleting everyting.

Reply:
would you rephrase it?

Reply:
yes, it sounds confusing. maybe 
```suggestion
  - Write the correct input and output channels and channel structure. This will try to reuse the information existing in your `meta.yml`, and only change the structure of the file if needed. Make sure to check that the added changes are correct and update `description`, `pattern` and `type` values accordingly.
```